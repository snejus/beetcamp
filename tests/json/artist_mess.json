{
  "@context": "https://schema.org",
  "@id": "https://psykovsky.bandcamp.com/album/ksolntsu",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 1199755283
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 1
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "various"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://psykovsky.bandcamp.com/album/ksolntsu",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 134886448
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 250137266
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 1199755283
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a1199755283_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "Ksolntsu"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/ela-na-pame",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/stone-sea",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/so-we-sailed-till-we-found",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/doors-of-perception",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/variant-on-the-right",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/call-of-beauty",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/many-many-krishnas",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/prem-i-um",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/now-here-nowhere",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/holy-black-little-lark",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/worlds-of-wisdom",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/pc-transmission",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/rs-lightmusic",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/ksolntsu",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://psykovsky.bandcamp.com/track/dadme-albricios-hijos-deva",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@type": "MusicGroup",
    "name": "Psykovsky & Friends"
  },
  "copyrightNotice": "Various",
  "dateModified": "12 Jul 2018 19:50:45 GMT",
  "datePublished": "12 Feb 2015 00:00:00 GMT",
  "image": "https://f4.bcbits.com/img/a1199755283_10.jpg",
  "keywords": [
    "Electronic",
    "Experimental",
    "Nanotech",
    "psytrance",
    "Niue"
  ],
  "mainEntityOfPage": "https://psykovsky.bandcamp.com/album/ksolntsu",
  "name": "Ksolntsu",
  "numTracks": 15,
  "publisher": {
    "@id": "https://psykovsky.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 250137266
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_download_codes",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 1920
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 21724693
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 1920
      }
    ],
    "description": "a maverick electronic music producer, since 2000 Psykovsky has worked with a number of different solo and collaborative projects, and is nowadays acknowledged as one of the most auspicious psytrance artists - check out Psykovsky's alternative digital audio-visual releases at https://www.youtube.com/channel/UCu1BnVhnFn0pcVZnkwmOAZQ",
    "foundingLocation": {
      "@type": "Place",
      "name": "Niue"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0021724693_10.jpg",
    "name": "Psykovsky",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "music",
        "url": "https://psykovsky.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://psykovsky.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/ela-na-pame",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3936209160
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Orestis"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H08M38S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/ela-na-pame",
          "name": "Ela Na Pame"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/stone-sea",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1697462790
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Luuli"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H11M13S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/stone-sea",
          "name": "Stone Sea"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/so-we-sailed-till-we-found",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 100558700
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Spiral"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H07M34S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/so-we-sailed-till-we-found",
          "name": "So We Sailed Till We Found"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/doors-of-perception",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 64137153
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Kasatka"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H07M53S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/doors-of-perception",
          "name": "Doors Of Perception"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/variant-on-the-right",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2945940491
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Spiral & Seeasound"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H12M16S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/variant-on-the-right",
          "name": "Variant On The Right"
        },
        "position": 5
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/call-of-beauty",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 249961523
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "attribution_non_commercial_share_alike"
            }
          ],
          "copyrightNotice": "Attribution Non Commercial Share Alike",
          "duration": "P00H12M49S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/call-of-beauty",
          "name": "Call Of Beauty"
        },
        "position": 6
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/many-many-krishnas",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1039297523
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Orestis & Jobaba"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H12M09S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/many-many-krishnas",
          "name": "Many Many Krishnas"
        },
        "position": 7
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/prem-i-um",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2584951270
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Kashyyyk & Arcek"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M49S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/prem-i-um",
          "name": "Prem I Um"
        },
        "position": 8
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/now-here-nowhere",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1788081328
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Arcek"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H09M17S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/now-here-nowhere",
          "name": "Now Here Nowhere"
        },
        "position": 9
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/holy-black-little-lark",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3407782947
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "attribution_non_commercial_share_alike"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Maleficium & Seeasound"
          },
          "copyrightNotice": "Attribution Non Commercial Share Alike",
          "duration": "P00H18M07S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/holy-black-little-lark",
          "name": "Holy Black / Little Lark",
          "recordingOf": {
            "@type": "MusicComposition",
            "lyrics": {
              "@type": "CreativeWork",
              "text": "Little lark\r\n\r\nall alone\r\n\r\ncannot find his way\r\n\r\nThrough the wheat fields\r\n\r\nhe will wander\r\n\r\nsearching all the day\r\n\r\n\"Oh mother!\r\n\r\nPlease come find me!”\r\n\r\nHe calls to the sky\r\n\r\nThe wheat\r\n\r\nrustling ‘round him\r\n\r\nis the only reply\r\n\r\nHe’s forgotten\r\n\r\nhow to find his way\r\n\r\nback to the nest\r\n\r\nLittle lark\r\n\r\nall alone now\r\n\r\nin the fields he rests\r\n\r\nThrough the dark\r\n\r\nthe fox hounding\r\n\r\nin the moonlight\r\n\r\nAfter sunset\r\n\r\nall that’s left is\r\n\r\nforlorn twilight"
            }
          }
        },
        "position": 10
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/worlds-of-wisdom",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 386177510
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M48S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/worlds-of-wisdom",
          "name": "Worlds Of Wisdom"
        },
        "position": 11
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/pc-transmission",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1401461526
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H09M21S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/pc-transmission",
          "name": "PC Transmission"
        },
        "position": 12
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/rs-lightmusic",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3234798938
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "attribution_non_commercial_share_alike"
            }
          ],
          "copyrightNotice": "Attribution Non Commercial Share Alike",
          "duration": "P00H06M51S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/rs-lightmusic",
          "name": "RS Lightmusic"
        },
        "position": 13
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/ksolntsu",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1634779854
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Psykovsky & Quip Tone Beatz & Flish"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H09M15S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/ksolntsu",
          "name": "Ksolntsu"
        },
        "position": 14
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://psykovsky.bandcamp.com/track/dadme-albricios-hijos-deva",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1223215058
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "byArtist": {
            "@type": "MusicGroup",
            "name": "Birds Of Praise"
          },
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H10M23S",
          "mainEntityOfPage": "https://psykovsky.bandcamp.com/track/dadme-albricios-hijos-deva",
          "name": "Dadme albricios hijos d'Eva"
        },
        "position": 15
      }
    ],
    "numberOfItems": 15
  }
}
