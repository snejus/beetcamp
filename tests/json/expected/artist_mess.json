[
  {
    "album": "Ksolntsu",
    "album_id": "https://psykovsky.bandcamp.com/album/ksolntsu",
    "albumdisambig": null,
    "albumstatus": "Official",
    "albumtype": "album",
    "albumtypes": "album",
    "artist": "Various Artists",
    "artist_credit": null,
    "artist_id": "https://psykovsky.bandcamp.com",
    "artist_sort": null,
    "asin": null,
    "catalognum": "",
    "comments": null,
    "country": "NU",
    "data_source": "bandcamp",
    "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
    "day": 12,
    "discogs_albumid": null,
    "discogs_artistid": null,
    "discogs_labelid": null,
    "genre": "experimental, psytrance",
    "label": "Psykovsky",
    "language": null,
    "media": "Digital Media",
    "mediums": 1,
    "month": 2,
    "original_day": null,
    "original_month": null,
    "original_year": null,
    "releasegroup_id": null,
    "releasegroupdisambig": null,
    "script": null,
    "style": "electronic",
    "tracks": [
      {
        "arranger": null,
        "artist": "Psykovsky & Orestis",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 1,
        "initial_key": null,
        "length": 518,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 1,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Ela Na Pame",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/ela-na-pame",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Luuli",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 2,
        "initial_key": null,
        "length": 673,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 2,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Stone Sea",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/stone-sea",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Spiral",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 3,
        "initial_key": null,
        "length": 454,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 3,
        "medium_total": 15,
        "release_track_id": null,
        "title": "So We Sailed Till We Found",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/so-we-sailed-till-we-found",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Kasatka",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 4,
        "initial_key": null,
        "length": 473,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 4,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Doors Of Perception",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/doors-of-perception",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Spiral & Seeasound",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 5,
        "initial_key": null,
        "length": 736,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 5,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Variant On The Right",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/variant-on-the-right",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Friends",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 6,
        "initial_key": null,
        "length": 769,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 6,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Call Of Beauty",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/call-of-beauty",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Orestis & Jobaba",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 7,
        "initial_key": null,
        "length": 729,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 7,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Many Many Krishnas",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/many-many-krishnas",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Kashyyyk & Arcek",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 8,
        "initial_key": null,
        "length": 409,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 8,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Prem I Um",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/prem-i-um",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Arcek",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 9,
        "initial_key": null,
        "length": 557,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 9,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Now Here Nowhere",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/now-here-nowhere",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Maleficium & Seeasound",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 10,
        "initial_key": null,
        "length": 1087,
        "lyricist": null,
        "lyrics": "Little lark\n\nall alone\n\ncannot find his way\n\nThrough the wheat fields\n\nhe will wander\n\nsearching all the day\n\n\"Oh mother!\n\nPlease come find me!”\n\nHe calls to the sky\n\nThe wheat\n\nrustling ‘round him\n\nis the only reply\n\nHe’s forgotten\n\nhow to find his way\n\nback to the nest\n\nLittle lark\n\nall alone now\n\nin the fields he rests\n\nThrough the dark\n\nthe fox hounding\n\nin the moonlight\n\nAfter sunset\n\nall that’s left is\n\nforlorn twilight",
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 10,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Holy Black / Little Lark",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/holy-black-little-lark",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Friends",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 11,
        "initial_key": null,
        "length": 408,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 11,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Worlds Of Wisdom",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/worlds-of-wisdom",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Friends",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 12,
        "initial_key": null,
        "length": 561,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 12,
        "medium_total": 15,
        "release_track_id": null,
        "title": "PC Transmission",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/pc-transmission",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Friends",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 13,
        "initial_key": null,
        "length": 411,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 13,
        "medium_total": 15,
        "release_track_id": null,
        "title": "RS Lightmusic",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/rs-lightmusic",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Psykovsky & Quip Tone Beatz & Flish",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 14,
        "initial_key": null,
        "length": 555,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 14,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Ksolntsu",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/ksolntsu",
        "work": null,
        "work_disambig": null
      },
      {
        "arranger": null,
        "artist": "Birds Of Praise",
        "artist_credit": null,
        "artist_id": "https://psykovsky.bandcamp.com",
        "artist_sort": null,
        "bpm": null,
        "composer": null,
        "composer_sort": null,
        "data_source": "bandcamp",
        "data_url": "https://psykovsky.bandcamp.com/album/ksolntsu",
        "disctitle": null,
        "genre": null,
        "index": 15,
        "initial_key": null,
        "length": 623,
        "lyricist": null,
        "mb_workid": null,
        "media": "Digital Media",
        "medium": 1,
        "medium_index": 15,
        "medium_total": 15,
        "release_track_id": null,
        "title": "Dadme albricios hijos d'Eva",
        "track_alt": null,
        "track_id": "https://psykovsky.bandcamp.com/track/dadme-albricios-hijos-deva",
        "work": null,
        "work_disambig": null
      }
    ],
    "va": true,
    "year": 2015
  }
]
